src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -34,6 +35,7 @@ def __init__(self, app, api, config, mount):
         self.reqmgr_db_service = RequestDBWriter(self.reqmgr_db, couchapp = "ReqMgr")
         # this need for the post validtiaon 
         self.reqmgr_aux_db = api.db_handler.get_db(config.couch_reqmgr_aux_db)
+        self.gq_service = WorkQueue("%s/%s" % (config.couch_host, config.couch_workqueue_db))
```

Comment:
Minor comment, but :) why do you append these variables? Can't you simply pass the host and db to be used?
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Services/WorkQueue/WorkQueue.py#L13